cmake/targets/gvsoc/PulpTarget.cmake

```diff
@@ -36,6 +36,7 @@ MACRO(ADD_EXECUTABLE_GVSOC_PULP_INTERNAL TARGET_NAME ADD_PLATFORM_FILES)
             -T ${GVSOC_TARGET_LIB_DIR}/link.ld \
             "
         )
+        SET(ARA_MACRO_ALREADY_EXECUTED ON)
```

Comment:
Previous lines still mention `GVSOC_PULP_MACRO_ALREADY_EXECUTED ` is that correct?